Tests/Kernels/Cosmos.Compiler.Tests.Bcl/System/Text/StringBuilderTest.cs

```diff
@@ -38,7 +37,7 @@ public static void Execute()
             sb.Clear();
 
             // Capacity does not change after Clear
-            Assert.IsTrue(sb.Capacity == 32, "StringBuilder.Capacity after Clear is wrong)");
+            Assert.IsTrue(sb.Capacity == 26, "StringBuilder.Capacity after Clear is wrong");
```

Comment:
This is wrong by specifications StringBuilder.Clear() should not change the Capacity property:
https://docs.microsoft.com/en-us/dotnet/api/system.text.stringbuilder.clear?view=netframework-4.7.2

but it is not a Cosmos issue really but it seems a Net Core 2.1 bug

Reply:
Probably their docs shouldn't be so specific... `Capacity` is always implementation specific, and we shouldn't really rely on it for testing IMO, the only thing we can test is that `Capacity >= Length`.